app/views/posts/index.html.erb

```diff
@@ -27,6 +28,8 @@
             </div>
           <% end %>
         </div>
+        <a href="<%= @user.id%>/posts/" class="button">Add Post</a>
```

Comment:
- [ ] [REQUIRED] Great job with all the implementations @edubew , your implementations work well except for the routing error I get when I click on the `Add post` button in an attempt to create a post in the form.
### To fix this:
- In order to access an individual post, use:
``` ruby
<%= link_to  user_post_path(@user, @post) do %>   
```
Let's also ensure the `Add Post` button is not overlapping anymore in order to have a more professional design.
<img width="671" alt="Screenshot 2023-06-17 at 21 32 25" src="https://github.com/edubew/blog-box/assets/58692590/392b4ca0-616c-498c-8b67-45b32aa73b25">

<img width="561" alt="Screenshot 2023-06-17 at 21 36 17" src="https://github.com/edubew/blog-box/assets/58692590/40c32ed1-b0ab-4af4-afea-991588b0637d">

Reply:
Hello @Whoistolu ,
Thank you for your invaluable feedback and insights on correcting the error. The button now works as it should. I really do appreciate the support you have provided